modules/ebi-metagenomics/interproscan/main.nf

```diff
@@ -25,7 +30,7 @@ process INTERPROSCAN {
     def args = task.ext.args ?: ''
     def prefix = task.ext.prefix ?: "${meta.id}"
 
-    def appl = "-appl TIGRFAM,FunFam,SFLD,PANTHER,Gene3D,Hamap,ProSiteProfiles,Coils,SMART,CDD,PRINTS,PIRSR,ProSitePatterns,AntiFam,Pfam,MobiDBLite"
+    def appl = "-appl TIGRFAM,SFLD,SUPERFAMILY,Gene3D,Hamap,Coils,ProSiteProfiles,SMART,CDD,PRINTS,PIRSF,ProSitePatterns,PfamA,ProDom,MobiDBLite,SignalP_GRAM_POSITIVE,SignalP_GRAM_NEGATIVE,SignalP_EUK,Phobius,TMHMM"
     if ( args.contains("-appl") ) {
```

Comment:
Some of the -appl arguments were creating errors for me on Codon. If I remember correctly Java erros. Can you run a fast one on Codon with all enabled to make sure this is not the case any more?

Reply:
Tested this with this version of the database: https://ftp.ebi.ac.uk/pub/software/unix/iprscan/5/5.62-94.0/ 

no errors anymore